torchgeo/datasets/idtrees.py

```diff
@@ -567,25 +567,26 @@ def normalize(x: Tensor) -> Tensor:
 
         return fig
 
-    def plot_las(self, index: int, colormap: Optional[str] = None) -> Any:
+    def plot_las(
+        self, index: int, colormap: Optional[str] = None
+    ) -> "pyvista.Plotter":  # type: ignore[name-defined] # noqa: F821
```

Comment:
I would add some `**kwargs` here so that users can control the `Plotter` init params and the `add_mesh` params.

Reply:
If switching to the `.plot()` API, these can all be captured and `**kwargs` and passed along